### PR TITLE
{cfd} [intel/2017a] OpenFOAM 4.1 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
OpenFOAM is stuck to Qt4 (and so are all of its deps).

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
nic69 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/20096f14721b1daf400860ed265ac4f2 for a full test report.

**Comment by boegel** [comment]
@wpoely86 please sync with `develop` & fix merge conflict?

File: easybuild/easyconfigs/q/Qt5/Qt5-5.8.0-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ('pkg-config', '0.29.2'),
+]
+
+# qtgamepad needs recent kernel/libevdev (fails on RHEL 6.x)
+configopts = '-skip qtgamepad'
+
+dependencies = [
+    ('GLib', '2.52.0'),
+    ('libpng', '1.6.29'),
+    ('X11', '20170314'), #, '', ('GCCcore', '6.3.0')),
```
Comment (boegel) [inline easybuild/easyconfigs/q/Qt5/Qt5-5.8.0-intel-2017a.eb]: @wpoely86 please remove commented out bit?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a6ae4b2b5c11df7fb5f3c3de8b0d75bb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2144.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9b37340a7e4657003bb3c3fc3e0dc72b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**